### DISCUSSION TITLE
Is there an export option for Pages, and what's the runway looking like?

### DISCUSSION THREAD

**Discussion by Gizmotronn** [makeplane/plane#3347] [Q&A]
My team are planning on moving away from Notion and have identified Pages as a good alternative, as we're already in the process of migration from Jira to Plane. However, I'm currently ensuring that all documents on our Plane instance are also added to our Notion as we've experienced data loss & other issues (e.g. the github integration) with Plane. Is there an export/backup option for our pages? What's the setup for pages on the self-hosted versions?

Is pages going to be updated significantly this year (i.e. integrations, better embeds, more styling options, databases/tables)? I understand it's an open-source product, but I'd be willing to pay for something like Pages, it's just not quite feature-rich enough yet for our team to fully migrate towards.

**Comment by srinivaspendem** [comment]
@Gizmotronn thanks for starting this discussion. Pages is a new feature we worked on to understand if there's significant interest from our community in adopting this in collaboration workflows along with managing issues. I would like to further talk to you to understand how you use notion and why you want to migrate. Would you be willing to talk to me about this? Here's a link to book a slot with me - https://app.cal.com/srinivas-pendem/product-research
